src/Avalonia.Labs.Controls/SKCanvasView/SKPaintSurfaceEventArgs.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SkiaSharp;
+using System;
+
+namespace Avalonia.Labs.Controls;
+
+public sealed class SKPaintSurfaceEventArgs : EventArgs
+{
+    public SKPaintSurfaceEventArgs(SKSurface surface, SKImageInfo info)
+        : this(surface, info, info)
+    {
+    }
+
+    public SKPaintSurfaceEventArgs(SKSurface surface, SKImageInfo info, SKImageInfo rawInfo)
```

Comment:
IMO we should provide an SKCanvas instance there and make the rest to be optional.
We might want to have a mode that utilizes SKPictureRecorder and custom composition visual to make SKCanvasView to be hardware-accelerated even on platforms where we don't provide GPU surface interop.

Reply:
I would like to maintain compatibility with https://github.com/mono/SkiaSharp/blob/25e70a390e2128e5a54d28795365bf9fdaa7161c/source/SkiaSharp.Views/SkiaSharp.Views.Shared/SKPaintSurfaceEventArgs.cs for quick porting.